mobile/lib/db/files_db.dart

```diff
@@ -106,6 +106,7 @@ class FilesDB {
   static Future<sqlite3.Database>? _ffiDBFuture;
   static Future<sqlite_async.SqliteDatabase>? _sqliteAsyncDBFuture;
 
+  @Deprecated("Use ffiDB instead (sqlite_async)")
```

Comment:
Should prompt to use sqliteAsyncDB instead?

Reply:
Yes! Will fix and create a PR.